presto-hive/src/test/sql/create-test-hive13.sql

```diff
@@ -82,7 +82,7 @@ SELECT * FROM presto_test_types_textfile
 ;
 
 
--- Parquet is missing TIMESTAMP and BINARY, and for some reason
```

Comment:
This script `hive13` is for testing Hive 0.13. Timestamp is actually not supported in that version:
```
FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.DDLTask. java.lang.UnsupportedOperationException: Parquet does not support timestamp. See HIVE-6384